WinImageBuilder.psm1

```diff
@@ -64,6 +64,13 @@ class PathShouldExistAttribute : System.Management.Automation.ValidateArgumentsA
     }
 }
 
+function Write-Log {
+    Param($messageToOut)
+
+    Write-Host $messageToOut (Get-Date)
+    $messageToOut | Out-File "export.txt" -Append;

```

Comment:
We should have a second parameter with the name of the file were we want to log the process. If the parameter is NULL, then the user chose to not log the process.